pkg/ddc/thin/transform_fuse.go

```diff
@@ -179,11 +180,33 @@ func (t *ThinEngine) parseFuseOptions(runtime *datav1alpha1.ThinRuntime, profile
 	for k, v := range runtime.Spec.Fuse.Options {
 		options[k] = v
 	}
-	// option in dataset will cover option in runtime
+
+	// option in dataset sharedOptions will cover option in runtime
+	for k, v := range dataset.Spec.SharedOptions {
+		options[k] = v
+	}
+
+	// option in dataset will cover sharedOptions and option in runtime
 	for k, v := range dataset.Spec.Mounts[0].Options {
 		// support only one mountpoint
 		options[k] = v
 	}
+
+	for _, encryptOption := range dataset.Spec.SharedEncryptOptions {
+		key := encryptOption.Name
+		secretKeyRef := encryptOption.ValueFrom.SecretKeyRef
+		secret, err := kubeclient.GetSecret(t.Client, secretKeyRef.Name, t.namespace)
+		if err != nil {
+			t.Log.Info("can't get the secret",
+				"namespace", t.namespace,
+				"name", t.name,
+				"secretName", secretKeyRef.Name)
+			return "", err
+		}
+		val := secret.Data[secretKeyRef.Key]
+		options[key] = string(val)
+	}
+
```

Comment:
Can we reuse the `genUFSMountOptions` func to replace this code block?

Reply:
Plus, I think it can be a future TODO to make `genUFSMountOptions` a utility function for all runtime transformation. For now, these functions from different engines share the same implementation

Reply:
ok